artifacts/otg.proto

```diff
@@ -16648,161 +16659,131 @@ message UpdateConfigRequest {
 
 
 message SetCaptureStateRequest {
   CaptureState capture_state = 1;
 }
 message SetCaptureStateResponse {
-  optional ResponseWarning status_code_200 = 1;
-  optional ResponseError status_code_400 = 2;
-  optional ResponseError status_code_500 = 3;
+  Warning warning = 1;
 }
 
 
 message UpdateFlowsRequest {
   FlowsUpdate flows_update = 1;
 }
 message UpdateFlowsResponse {
-  optional Config status_code_200 = 1;
-  optional ResponseError status_code_400 = 2;
-  optional ResponseError status_code_500 = 3;
+  Config config = 1;
 }
 
 
 message SetRouteStateRequest {
   RouteState route_state = 1;
 }
 message SetRouteStateResponse {
-  optional ResponseWarning status_code_200 = 1;
-  optional ResponseError status_code_400 = 2;
-  optional ResponseError status_code_500 = 3;
+  Warning warning = 1;
 }
 
 
 message SendPingRequest {
   PingRequest ping_request = 1;
 }
 message SendPingResponse {
-  optional PingResponse status_code_200 = 1;
-  optional ResponseError status_code_400 = 2;
-  optional ResponseError status_code_500 = 3;
+  PingResponse ping_response = 1;
 }
 
 
 message SetProtocolStateRequest {
   ProtocolState protocol_state = 1;
 }
 message SetProtocolStateResponse {
-  optional ResponseWarning status_code_200 = 1;
-  optional ResponseError status_code_400 = 2;
-  optional ResponseError status_code_500 = 3;
+  Warning warning = 1;
 }
 
 
 message SetDeviceStateRequest {
   DeviceState device_state = 1;
 }
 message SetDeviceStateResponse {
-  optional ResponseWarning status_code_200 = 1;
-  optional ResponseError status_code_400 = 2;
-  optional ResponseError status_code_500 = 3;
+  Warning warning = 1;
 }
 
 
 message GetMetricsRequest {
   MetricsRequest metrics_request = 1;
 }
 message GetMetricsResponse {
-  optional MetricsResponse status_code_200 = 1;
-  optional ResponseError status_code_400 = 2;
-  optional ResponseError status_code_500 = 3;
+  MetricsResponse metrics_response = 1;
 }
 
 
 message GetStatesRequest {
   StatesRequest states_request = 1;
 }
 message GetStatesResponse {
-  optional StatesResponse status_code_200 = 1;
-  optional ResponseError status_code_400 = 2;
-  optional ResponseError status_code_500 = 3;
+  StatesResponse states_response = 1;
 }
 
 
 message GetCaptureRequest {
   CaptureRequest capture_request = 1;
 }
 message GetCaptureResponse {
-  optional bytes status_code_200 = 1;
-  optional ResponseError status_code_400 = 2;
-  optional ResponseError status_code_500 = 3;
+  bytes response_bytes = 1;
 }
 
 message GetVersionResponse {
-  optional Version status_code_200 = 1;
-  optional ResponseError status_code_400 = 2;
-  optional ResponseError status_code_500 = 3;
+  Version version = 1;
 }
 
 
-  // Description missing in models
+// Description missing in models
+//
+// For all RPCs defined in this service, API Server SHOULD provide JSON
+// representation of `Error` message as an error string upon failure, ensuring
+// name of enum constants (instead of value) is present in the representation
```

Comment:
...(instead of value) is present in the representation --> maybe insert '.. is present for the 'kind' attribute ...

Reply:
done